### PR TITLE
Add documentation for 3D scatter plot & surface plot

### PR DESCRIPTION
**Pull request by phchen95**

### CODE REVIEW

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 1]
```diff
@@ -150,6 +150,38 @@
 #
 # .. image:: /_static/heatmap.png
 #
+# Visualize a 3D surface plot
+# --------------------------
+#
+# A 3D surface plot allows you to visualize the data points in a terrain like
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:156]:
```suggestion
# A 3D surface plot allows you to visualize the data points in a terrain-like
```

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 2]
```diff
@@ -150,6 +150,38 @@
...
+# surface in a 3D space powered by WebGL engine. It is a variation of a
+# heatmap as they share the same data structure. The difference is that the
+# surface plot visualize the heatmap values as the Z axis data, allowing you
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:159]:
```suggestion
# surface plot visualizes the heatmap values as the Z axis data, allowing you
```

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 3]
```diff
@@ -150,6 +150,38 @@
...
+# to zoom in and out, and rotate to view the surface in different angles. Like
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:160]:
```suggestion
# to zoom in and out, and rotate to view the surface from different angles. Like
```

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 4]
```diff
@@ -150,6 +150,38 @@
 #
 # .. image:: /_static/heatmap.png
 #
+# Visualize a 3D surface plot
+# --------------------------
+#
+# A 3D surface plot allows you to visualize the data points in a terrain like
+# surface in a 3D space powered by WebGL engine. It is a variation of a
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:157]:
```suggestion
# surface in a 3D space powered by the WebGL engine. It is a variation of a
```

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 5]
```diff
@@ -227,6 +259,40 @@
 #
 # .. image:: /_static/sankey.png
 #
+# Visualize a 3D scatter plot
+# --------------------------
+#
+# A 3D scatter plot allows you to visualize the data points in 3D space
+# powered by WebGL engine. You can zoom in and out, and rotate to view the
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:266]:
```suggestion
# powered by the WebGL engine. You can zoom in and out, and rotate to view the
```

File: doc/source/examples_source/00-basic/02-plottype.py [hunk 6]
```diff
@@ -227,6 +259,40 @@
...
+# data in different angles.
```
Comment (JennaPaikowsky) [inline doc/source/examples_source/00-basic/02-plottype.py:267]:
```suggestion
# data from different angles.
```

**Review by JennaPaikowsky — COMMENTED** [review]
Just a few minor suggestions

**Review by margalva — APPROVED** [review]

**Pull request merged**